docs/boards/Board - RUSHCORE7.md

```diff
@@ -0,0 +1,101 @@
+# Board - RUSHCORE7
+
+The RUSHCORE7 described here:
+
+This board use the STM32F722RET6 microcontroller and have the following features:
```

Comment:
Now you are not referencing any of the pictures in here - if the pictures are not needed, please remove them from the pull request, or else add them to your board description.

Reply:
Not need yet.How should I delete it from the pull request?

Reply:
@mikeller Can you tell me how to delete it?Thanks

Reply:
Delete the files locally in your branch with `git rm <filename>`, then commit and push the changes.